connexion/validators/json.py

```diff
@@ -129,7 +129,7 @@ def _validate(self, body: dict):
             self.validator.validate(body)
         except ValidationError as exception:
             error_path_msg = format_error_with_path(exception=exception)
-            logger.error(
+            logger.info(
```

Comment:
For this error, I propose to use the `warning` level as the response body should adhere to the schema. So in that sense it is more a server error than a client error (i.e. something for the application to fix).
What do you think?

Reply:
Thanks for reviewing my proposal, really appreciate it! 👍 

The reason I put it to `info` is, that is doesn't contribute any value:

* `NonConformingResponseBody` inherits from `InternalServerError` and results in a `500` response . Therefore it's  logged to `error` level in `ExceptionMiddleware`.
* `BadRequestProblem` results in a `400` response. Therefore it's logged to `warning` level in `ExceptionMiddleware`.

I think that the error handling should be responsible for deciding about log levels, not the validation logic itself.

Now that I write about it: I'd rather propose to delete both log statements from the json logger completely.

Reply:
Changed the log level accordingly.